src/main/java/level4.java

```diff
@@ -1,2 +1,75 @@
-package PACKAGE_NAME;public class level4 {
+import java.util.Scanner;
+
+public class level4 {
+    public static void main(String[] args){
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        String line;
+        Task[] tasks = new Task[100];
```

Comment:
The task class could also include the "done" and the "type" as well, ensuring better OOP practices as currently that information is segregated.